docs/x509/verification.rst

```diff
@@ -297,3 +323,182 @@ the root of trust:
             for server verification.
 
         :returns: An instance of :class:`ClientVerifier`
+
+.. class:: ExtensionPolicyBuilder
+
+    .. versionadded:: 44.0.0
+
+    An ExtensionPolicyBuilder provides a builder-style interface for constructing an 
+    :class:`ExtensionPolicy`.
+
+    .. staticmethod:: permit_all()
+
+        Creates an ExtensionPolicyBuilder initialized with a policy that does 
+        not put any constraints on a certificate's extensions. 
+
+        :returns: An instance of :class:`ExtensionPolicyBuilder`
+
+    .. staticmethod:: webpki_defaults_ca()
+
+        Creates an ExtensionPolicyBuilder initialized with a 
+        CA extension policy based on CA/B Forum guidelines.
+
+        This is the CA extension policy used by :class:`PolicyBuilder`.
+
+        :returns: An instance of :class:`ExtensionPolicyBuilder`
+
+    .. staticmethod:: webpki_defaults_ee()
+
+        Creates an ExtensionPolicyBuilder initialized with an
+        EE extension policy based on CA/B Forum guidelines.
+
+        This is the EE extension policy used by :class:`PolicyBuilder`.
+
+        :returns: An instance of :class:`ExtensionPolicyBuilder`
```

Comment:
@woodruffw right now our extension policies are basically "CABF, but with some things loosened where necessary in practice", is there a good way of articulating that for users?

Reply:
> @woodruffw right now our extension policies are basically "CABF, but with some things loosened where necessary in practice", is there a good way of articulating that for users?

Sorry, missed this ping originally!

Maybe something like "these extension policies are intended to be strictly consistent with CABF, which dictates the requirements for certificates on the public web. However, we make limited exceptions to CABF where necessary in practice, i.e. behaviors that CABF forbids but are nonetheless present in the current Web PKI."

But maybe that's too much of a mouthful.

Reply:
We can put it in a glossary entry or something

Reply:
I'm not quite sure how to formulate this as a glossary entry. What would be the entry name? I would appreciate if someone could write out the full text of the entry here (or just made a separate PR for this).